src/Components/Auth/Login.tsx

```diff
@@ -109,7 +109,12 @@ export const Login = (props: { forgot?: boolean }) => {
           window.location.pathname === "/" ||
           window.location.pathname === "/login"
         ) {
-          window.location.href = "/facility";
+          const redirectParam = new URLSearchParams(window.location.search).get(
+            "redirect"
+          );
+          redirectParam
+            ? (window.location.href = window.location.origin + redirectParam)
```

Comment:
This is still vulnerable to open redirection attack,

POC (Access while logged out): [deploy-preview-6495--care-egov-staging.netlify.app/?redirect=.my.subdomain.attacker.com/virus](https://deploy-preview-6495--care-egov-staging.netlify.app/?redirect=.my.subdomain.attacker.com/virus)